decorators/placeholder.js

```diff
@@ -6,18 +6,58 @@ var _ = require('lodash'),
   addComponentHandler = require('../services/components/add-component-handler');
 
 /**
+ *
+ * @param {Array} groupItems
+ * @param {string} propName
+ * @returns {object}
+ */
+function findPropInFieldGroup(groupItems, propName) {
+  return _.find(groupItems, groupProp => _.get(groupProp, '_schema._name') === propName) || null;
```

Comment:
I don't think you need to do `|| null` explicitly. doesn't `_.find` return null if nothing's found?